### ISSUE TITLE
Writing style guidelines

### ISSUE THREAD

**Issue by dngray** [privacyguides/privacyguides.org#908]
To make Privacy Guides a high quality resource I propose that we have some guidelines, that how content should be presented.

I like having advanced articles of the site, but I think needs to be a bit like an onion, in respect to being able to peel off entry level layers and get some usefulness out of it without being bombarded with very in-depth specifics that might turn off new readers.

This would allow for entry level articles to self cite more complex reasoning, for issues discussed.

I described that in https://github.com/privacyguides/privacyguides.org/issues/834#issuecomment-1087354314

We might want to get a guideline, this is a good one I think https://www.gov.uk/guidance/content-design/writing-for-gov-uk.

I think in general this would be an improvement, as it would prevent highly specific information being on the surface https://github.com/privacyguides/privacyguides.org/pull/904/files/7be16078d8d7e6fb8eb0469fb59da210dac7dbc1#r842391988

This issue is one i want to get codified before we begin outreach for translations https://github.com/privacyguides/privacyguides.org/discussions/30

I think it will help translators a lot.

**Comment by tommytran732** [comment]
The page is extremely inconsistent. Should additional information be inside of the cards like the Android or Linux page or by inside of the cards like the rest of the site?

I think its best to just do it like the Android/Linux page, so we can have a brief discussion in the card, but more in depth information can be put outside the card and the user can ignore what's outside of the cards if they are not interested.

**Comment by freddy-m** [comment]
Would this require a style guide?

**Comment by jonaharagon** [comment]
> I think its best to just do it like the Android/Linux page

Yes 👍

**Comment by jonaharagon** [comment]
https://www.plainlanguage.gov/guidelines/ is a similar set of guidelines in American English.

**Comment by elitejake** [comment]
Also check out Wikipedia''s [Manual of Style](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style).

**Comment by Razac-elda** [comment]
After a quick look at those guidelines I think @elitejake found the right one, the UK and US ones are good for general article advice. Wikipedia introduces concepts for more advanced styles and, being a widley popular website, users will find it more familiar to read.

**Comment by dngray** [comment]
Another one that wouldn't hurt is https://docs.microsoft.com/en-us/style-guide/procedures-instructions/writing-step-by-step-instructions

**Comment by elitejake** [comment]
We shouldn't follow these guidelines as-is, because they don't make use of Markdown. 

For example, Microsoft's guide state that we should use __bold text__ for commands instead of code blocks.

**Comment by qua3k** [comment]
is this the right place to ask for line wrap please

**Comment by dngray** [comment]
> is this the right place to ask for line wrap please

Been thinking about that myself actually.
I'm actually not sure why we never did do line wrapping

**Comment by jonaharagon** [comment]
I would be extremely against traditional hard-wrapping at an arbitrary set column size, but I'd consider reformatting to [one sentence per line](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) if we wanted, which would probably have great VCS benefits for our Git-oriented workflow as a bonus.

Another thought on the plain-text readability side of things, we probably should consider [reference-style links](https://www.markdownguide.org/basic-syntax/#reference-style-links) rather than inline links, for all of our links but especially duplicate links on the same page.

**Comment by lx-is** [comment]
I think we should try to avoid using "user/users" for more descriptive terms. As far as I know, this is what most companies with documentation writing styles practice. Relevant discussion: #1080

I found the [GitHub Explore style guide](https://github.com/github/explore/blob/main/docs/styleguide.md) to be laid out nicely. Could be used for some inspiration.

**Comment by dngray** [comment]
> I think we should try to avoid using "user/users" for more descriptive terms. As far as I know, this is what most companies with documentation writing styles practice. Relevant discussion: #1080

Only exception to this would be where it relates to some terminology. so "user permissions" for example.

**Comment by jonaharagon** [comment]
In response to my own comment https://github.com/orgs/privacyguides/discussions/1080#discussioncomment-2619033:

> Then we would need to determine the formality of the site's writing, whether we should stick with third-person or use words like "you"

I think referring to the reader directly is a good thing, we're giving advice to people, not writing an academic study of these tools. I'm working on a PR replacing "user" language now.

**Comment by jonaharagon** [comment]
Started drafting a page at https://github.com/privacyguides/privacyguides.org/wiki/Writing-Style

Standardizing on:

- American English
- [plainlangauge.gov](https://www.plainlanguage.gov/guidelines/) guidelines for writing and tone
- APA Style rules for grammar and style

**Comment by jonaharagon** [comment]
I think we can work out details separately, this is basically done.
